sw/device/lib/drivers/rv_plic/rv_plic.c

```diff
@@ -12,20 +46,33 @@
 #include "mmio.h"
 
 #include "rv_plic_regs.h"  // Generated.
+#include "core_v_mini_mcu.h"
```

Comment:
you are actually adding this twice, as it is already added into rv_plic_structc.h, right?